.gitignore

```
@@ -2,4 +2,7 @@
 docker-compose_windows.yml
 
 #Visual studio folder
-**/.vs
+**/.vs
+
+#Webstorm folder
+.idea/*
```

This is more flexible
```suggestion
**/.idea
```